src/decoding.cc

```diff
@@ -170,6 +239,15 @@ namespace ctranslate2 {
       attention->resize(batch_size);
     }
 
+    // TODO(scotfang): Instead of initializing and empty vector, use unique_ptr.
```

Comment:
I don't think initializing an empty vector will cause any performance issue, so you can probably remove this comment.

Reply:
done